.github/workflows/sonarcloud-analysis.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       - name: Set up JDK 17
```

Comment:
@sathwikhbhat I believe this is the last change: please update this label to say 21 instead of 17. Then I think all is ready to go.